docs/bugs_and_glitches.md

```diff
@@ -1881,6 +1882,34 @@ This bug can affect Mew or Pokémon other than Ditto that used Transform via Mir
 ```
 
 
+## Having the Dude catch a Pokémon while your boxes and party are full shows the box is full message
```

Comment:
How about: "If your party and current PC box are full during the Dude's catching tutorial, he can't use a Poké Ball"

Reply:
Since the message doesn't simply print and close, but causes further graphical glitching and potential freezing: "If your party and current PC box are full during the Dude's catching tutorial, his Poké Ball can't be used and may crash the game" (the relevant description for Crystal as opposed to G/S)

Reply:
In the German version, the graphical glitches will either freeze (VC) or work as an infinite money trick (real GB). Could that be added to your description?

"If your party and current PC box are full during the Dude's catching tutorial, his Poké Ball can't be used and may either crash the game or give a lot of money"

Reply:
If we ever get German as a build option, the bugs and glitches can be updated along with the code.

Reply:
I doubt describing what the effects of undefined behavior are very relevant.

Reply:
Omitted the undefined behavior from the description.

Reply:
Omitted the undefined behavior from the description as per @Rangi42's suggestion.